sky/templates/gcp-ray.yml.j2

```
@@ -96,10 +102,10 @@ available_node_types:
         - onHostMaintenance: TERMINATE  # Required for GPU-attached VMs.
   {%- endif %}
 {%- endif %}
-{% if num_nodes > 1 %}
+{% if num_nodes - num_reserved_nodes > 1 %}
```

Sorry, my bad. This seems need to be `>= 1` instead.
```suggestion
{% if num_nodes - num_reserved_nodes >= 1 %}
```